.husky/pre-commit

```diff
@@ -9,7 +9,7 @@ echo 'Running pre-commit hook...'
 chronic typos .
 
 # lint and format for core contracts and typescript codes
-(cd core && chronic pnpm lint && chronic pnpm format)
+(cd core && chronic pnpm lint)
```

Comment:
is there anything break we remove pnpm format here? seems will just run `forge fmt` so should be fine?

Reply:
I found it a bit confusing for a pre-commit hook to alter the code automatically. I disabled it for that reason.

I would rather we enable "Format on Save" option in VS code. For Solidity:
```
{
		"[solidity]": {
			"editor.formatOnSave": true,
			"editor.defaultFormatter": "JuanBlanco.solidity" 
		},
		"solidity.formatter": "forge",
}
```